pkg/secrets/s3.go

```diff
@@ -23,37 +23,37 @@ type S3Secret struct {
 }
 
 type S3Decrypter struct {
-	secretConfig string
+	encryptedSecret string
 }
 
-func NewS3Decrypter(secretConfig string) *S3Decrypter {
-	return &S3Decrypter{secretConfig}
+func NewS3Decrypter(encryptedSecret string) *S3Decrypter {
+	return &S3Decrypter{encryptedSecret}
 }
 
 func (s3 *S3Decrypter) Decrypt() (string, error) {
-	s3Secret, err := parseS3SecretConfig(s3.secretConfig)
+	s3Secret, err := parseS3EncryptedSecret(s3.encryptedSecret)
 	if err != nil {
 		return "", err
 	}
-	secret, err := s3Secret.fetchSecret()
+	decrypted, err := s3Secret.fetchSecret()
```

Comment:
can't you just `return s3Secret.fetchSecret()`?

Reply:
yes. yes i can.